apis/config/v1beta1/defaults_test.go

```diff
@@ -624,6 +625,70 @@ func TestSetDefaults_Configuration(t *testing.T) {
 				},
 			},
 		},
+		"localQueue metrics should default to nil": {
+			original: &Configuration{
+				InternalCertManagement: &InternalCertManagement{
+					Enable: ptr.To(false),
+				},
+			},
+			want: &Configuration{
+				Namespace:         ptr.To(DefaultNamespace),
+				ControllerManager: defaultCtrlManagerConfigurationSpec,
+				InternalCertManagement: &InternalCertManagement{
+					Enable: ptr.To(false),
+				},
+				ClientConnection:             defaultClientConnection,
+				Integrations:                 defaultIntegrations,
+				QueueVisibility:              defaultQueueVisibility,
+				MultiKueue:                   defaultMultiKueue,
+				ManagedJobsNamespaceSelector: defaultManagedJobsNamespaceSelector,
+			},
+		},
+		"if localQueueMetrics is set it should default to disabled with a nil lq selector": {
```

Comment:
I think you should also test the case when LQM is enabled but label selector is not specified.

Reply:
Sorry I got confused. I still think you should have a test case with the label Selector set.